cmd/ethereum_rust/ethereum_rust.rs

```diff
@@ -166,6 +166,17 @@ async fn main() {
             tracker.spawn(l2_operator);
             // let l2_prover = ethereum_rust_l2::start_prover().into_future();
             // tracker.spawn(l2_prover);
+        } else if #[cfg(feature = "dev")] {
+            use ethereum_rust_dev;
+
+            let authrpc_jwtsecret = std::fs::read(authrpc_jwtsecret).expect("Failed to read JWT secret");
+            let head_block_hash = {
```

Comment:
any reason you do this here instead of calling the appropriate rpc endpoint from the consensus client?

Reply:
As we have the state available it was easier to do this than to prepare a request and send it to the RPC client.